pkgs/by-name/po/powertabeditor/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+  ninja,
+  pkg-config,
+  doctest,
+
+  boost,
+  qt6,
+  nlohmann_json,
+  rtmidi,
+  pugixml,
+  minizip,
+
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "powertabeditor";
+  version = "2.0.21";
+
+  src = fetchFromGitHub {
+    owner = "powertab";
+    repo = "powertabeditor";
+    tag = finalAttrs.version;
+    hash = "sha256-mYFguVcF7Xb6rXIIMXAUzRYddlvQDRj7zu6on7GGGeA=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+    doctest
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    boost
+    qt6.qtbase
+    qt6.qttools
+    nlohmann_json
+    rtmidi
+    pugixml
+    minizip
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "View and edit guitar tablature";
+    homepage = "https://powertab.github.io/";
+    changelog = "https://github.com/powertab/powertabeditor/blob/${finalAttrs.src.rev}/CHANGELOG.md";
```

```suggestion
    changelog = "https://github.com/powertab/powertabeditor/blob/${finalAttrs.src.tag}/CHANGELOG.md";
```
Since you're using the new `tag` argument, `rev` is null -- you could also use `finalAttrs.version` instead

---

Huh. I thought they simply set `rev` to be `refs/tags/${tag}` as a polyfill - feels like an oversight to me

---

Nope, it's all done in a [local scope](https://github.com/NixOS/nixpkgs/blob/1434a1b1d4f9bf1da83375479be355494afd8478/pkgs/build-support/fetchgithub/default.nix#L67) :(

---

Cringe